nowcasting_dataset/config/model.py

```diff
@@ -151,6 +154,57 @@ class HRVSatellite(DataSourceMixin):
     hrvsatellite_meters_per_pixel: int = METERS_PER_PIXEL_FIELD
 
 
+class OpticalFlow(DataSourceMixin):
+    """Optical Flow configuration model"""
+
+    opticalflow_zarr_path: str = Field(
+        "",
+        description=(
+            "The satellite Zarr data to use. If in doubt, use the same value as"
+            " satellite.satellite_zarr_path."
+        ),
+    )
+    opticalflow_history_minutes: int = Field(
```

Comment:
you could use `DataSourceMixin` which has extra properties, and default values will be set from `InputData.default_forecast_minutes` and `InputData.default_history_minutes`

Reply:
done - I left in the description, as it was useful to read